examples/gallery/embellishments/scalebar.py

```diff
@@ -0,0 +1,112 @@
+     case **j** / **J** mean inside / outside of the map bounding box.
+   - **n**: Give normalized bounding box coordinates as *nx*\/\ *ny*.
+   - **x**: Give plot coordinates as *x*\/\ *y*.
+
+ - length: **+w**. Give a distance value, and, optionally a distance unit.
+   Choose from **e** (meters), **f** (feet), **k** (kilometers) [Default],
+   **M** (statute miles), **n** (nautical miles), or **u** (US survey feet).
+ - origin: **+c**\ [*slon*/]\ *slat*. Control where on the map the scale bar
+   applies. If **+c** is not given the reference point is used. If only
+   **+c** is appended the middle of the map is used. Note that *slon* is only
+   optional for projections with constant scale along parallels, e.g.,
+   Mercator projection.
+ - justify: **+j**. Set the anchor point. Specify a two-character (order
+   independent) code. Choose from vertical **T**\(op), **M**\(iddle), or
+   **B**\(ottom) and horizontal **L**\(eft), **C**\(entre), or **R**\(ight).
+ - offset: **+o**\ *offset* or **+o**\ *xoffset*/\ *yoffset*. Give either a
+   common shift or individual shifts in x (longitude) and y (latitude)
+   directions.
+ - height: Use :gmt-term:`MAP_SCALE_HEIGHT` via :func:`pygmt.config`.
+ - fancy style: **+f**. Get a scale bar that looks like train tracks.
+ - unit: **+u**. Add the distance unit given via **+w** to the single
+   distance values.
+ - label: **+l**. Add the distance unit given via **+w** as label. Append
+   text to get a customized label instead.
+ - alignment: **+a**. Set the label alignment. Choose from **t**\(op)
+   [Default], **b**\(ottom), **l**\(eft), or **r**\(ight).
+"""
+
+# %%
+import pygmt
+
+# Create a new Figure instance
+fig = pygmt.Figure()
+
+# Mercator projection with 10 centimeters width
+fig.basemap(region=[-45, -25, -15, 0], projection="M0/0/10c", frame=["WSne", "af"])
+
+# -----------------------------------------------------------------------------
+# Top Left: Add a plain scale bar
+# It is placed based on geographic coordinates (g) 42° West and 1° South,
+# applies at the reference point (+c is not given), and represents a
+# length (+w) of 500 kilometers
+fig.basemap(map_scale="g-42/-1+w500k")
+
+# -----------------------------------------------------------------------------
+# Top Right: Add a fancy scale bar
+# It is placed based on normalized bounding box coordinates (n)
+# Use a fancy style (+f) to get a scale bar that looks like train tracks
+# Add the distance unit (+u) to the single distance values
+fig.basemap(map_scale="n0.8/0.95+w500k+f+u")
+
+# -----------------------------------------------------------------------------
+# Bottom Left: Add a thick scale bar
+# Adjust the GMT default parameter MAP_SCALE_HEIGHT locally (the change applies
+# only to the code within the "with" statement)
+# It applies (+c) at the middle of the map (no location is appended to +c)
+# Without appending text, +l adds the distance unit as label
+with pygmt.config(MAP_SCALE_HEIGHT="20p"):
```

Comment:
`20p` is a little too thick. Maybe `10p` is better?
```suggestion
with pygmt.config(MAP_SCALE_HEIGHT="10p"):
```

Reply:
Yes, your are right. After reducing the width to 500 kilometers, we should also reduce the thickness of the scale bar.

Reply:
I also thought about combining the two subplots into one figure. After the change in commit https://github.com/GenericMappingTools/pygmt/pull/2822/commits/311cca7d2f9972f14e69c1ce8d99eb44140164a0 the gallery example is significantly shorter and clearer 🙂.